.github/workflows/docker-publish.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
           registry: docker.pkg.github.com
       - name: Push to GitHub Packages - Nightly
-        if: contains(github.ref, 'refs/head/main')
+        if: contains(github.ref, 'refs/head/main') || github.event.inputs.tag == ''
```

Comment:
this is effectively nullifying the first condition and will make this workflow run for any branch, which I don't think is what we want.

Reply:
Fixed